frappe/utils/html_utils.py

```diff
@@ -51,7 +51,7 @@ def clean_script_and_style(html):
 		s.decompose()
 	return frappe.as_unicode(soup)
 
-def sanitize_html(html, linkify=False):
+def sanitize_html(html, strip=False):
```

Comment:
changing method signature is a breaking change. need more information on why this is needed.